web/README.md

```diff
@@ -108,7 +113,52 @@ python manage.py test
 https://www.onlinetutorialspoint.com/django/django-model-view-template-mvt-overview.html
 
 ## Folder and file structure
-Where is what, and what does it do.
+
+As of now, the project consists of 6 folders, with 4 of them being django apps
+(`adminpage`, `authenticate`, `happenings` and `swiping`),
+one being a folder for util files (`util`) and the last one being the main folder (`web`).
+
+####Adminpage
+The `adminpage` app contains all pages related to the adminpage and the subpages. 
+The adminpage contains pages for curating events and creating/editing/deleting short descriptions, accessibility tags
+and event categories.
+
+The app has a `migration` folder for the database migrations, a `static` folder where css files are located and a 
+`templates` folder for the templates.
+migrations. The folder also contains files created upon django app creation and a `forms.py` file. 
+
+
+####Authenticate
+The `authenticate` app contains all pages and models related to users, login and registration.
+
+The app has a `migration` folder for the database migrations and a `template` folder for the templates.
```

Comment:
I don't think it is necessary to mention `migration` and `template` folders for each app since it will become very repetitive . Instead you can mention it in the beginning that every app contains `migration`, `template`  or other common folders.